src/libs/utils.js

```diff
@@ -89,3 +89,34 @@ export const signxcarchive = async (archiveFilePath, workspace = '/tmp/') => {
 
   return Promise.reject();
 };
+
+/**
+ * Fetch the values from keychain
+ *
+ * @param {object} keyNames The names of target keys in keychain
+ * @param {string} keyAccount The account registered with the keys
+ * @returns An object with the keychain name:value pairs
+ */
+export const fetchKeychainValue = async (keyNames, keyAccount) => {
+  var keyPairs = {};
+
+  // Modeify foreach function https://codeburst.io/javascript-async-await-with-foreach-b6ba62bbf404
+  async function asyncForEach(array, callback) {
```

Comment:
I feel like this could be simplified using a more declarative approach. When you get here stop by and we can hack on it together.